pyam/core.py

```diff
@@ -1284,6 +1287,48 @@ def export_meta(self, excel_writer, sheet_name='meta'):
         if close:
             excel_writer.close()
 
+    def to_datapackage(self, path, tmp='tmp'):
+        """Write object to a frictionless datapackage
```

Comment:
```suggestion
        """Write object to a frictionless Data Package
```

That's how they spell it in their docs.